sdks/python/apache_beam/__init__.py

```diff
@@ -77,6 +77,9 @@
 from apache_beam import coders
 from apache_beam import io
 from apache_beam import typehints
+from apache_beam import version
 from apache_beam.pipeline import Pipeline
 from apache_beam.transforms import *
 # pylint: enable=wrong-import-position
+
+__version__ = version.get_version()
```

Comment:
Most python libraries expose version at the root level so you can do something like

```
import xyz
xyz.__version__
```

That isn't true in python so I wanted to make that happen. Eg. https://github.com/pandas-dev/pandas/blob/master/pandas/__init__.py#L71